scripts/import_bch_flat.py

```diff
@@ -17,10 +17,13 @@
                         help='PostgreSQL connection string of form:\n\tdialect+driver://username:password@host:port/database\nExamples:\n\tpostgresql://scott:tiger@localhost/mydatabase\n\tpostgresql+psycopg2://scott:tiger@localhost/mydatabase\n\tpostgresql+pg8000://scott:tiger@localhost/mydatabase')
     parser.add_argument('-D', '--diag', action="store_true", default = False,
```

Comment:
Hey, if we're going to rename parameters (e.g., `filelist`), let's rename this either `--diagnostic` or perhaps the more common and descriptive `--dry_run`. I keep reading `diag` as "diagonal", as in "diagonal of a matrix", which is semi-plausible in this context but completely wrong.